Addressing a TODO about /functionSyntax (#2857)
dafny-lang/dafny: docs/DafnyRef/Types.md

}
```

Pre v4.0, a function is `ghost` by default, and cannot be called from non-ghost
code. To make it non-ghost, replace the keyword `function` with the two

> keywords "`function method`". From v4.0 on, a function is non-ghost by
> default and needs the `ghost` keyword to make it ghost.

Reviewer: A suggestion to make the code change more explicit as in the preceding sentence:
```suggestion
keywords "`function method`". From v4.0 on, a function is non-ghost by
default. To make it ghost, replace the keyword `function` with the two keywords "`ghost function`".
```
